packages/components/addon/components/hds/form/radio-card/index.hbs

```diff
@@ -0,0 +1,20 @@
+<label class={{this.classNames}} {{did-insert this.setAriaDescribedBy}}>
+  <div class="hds-form-radio-card__content">
+    {{yield (hash Icon=(component "flight-icon" size="24"))}}
+    {{yield (hash Label=(component "hds/form/radio-card/label"))}}
+    {{yield (hash Badge=(component "hds/badge"))}}
+    {{yield (hash Description=(component "hds/form/radio-card/description"))}}
+    {{yield (hash Generic=(component "hds/yield"))}}
+  </div>
+  <div class="hds-form-radio-card__control-wrapper">
+    <Hds::Form::Radio::Base
+      class="hds-form-radio-card__control"
+      @value={{@value}}
+      name={{@name}}
+      checked={{@checked}}
+      disabled={{@disabled}}
+      aria-describedby={{this.ariaDescribedBy}}
+      ...attributes
+    />
+  </div>
```

Comment:
Nit: Technically the label element should only contain phrasing content (inline-level content) so should not be used to contain div tags.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#technical_summary

Reply:
Yep 😬, aware of the spec, and while I'm not a supporter of `label`s wrapping `input`s I think in this very instance following [Melanie's prototype](https://codepen.io/melsumner/pen/oNEGJaw?editors=1100) has many benefits and reduces the complexity of this component (in terms of keeping element states in sync, etc).

Reply:
Hmm? It's fine for labels to wrap inputs. Just saying preferably we wouldn't use labels to wrap div tags since that's invalid. We could use span tags instead to make the markup validate.

Reply:
Cool, thanks! 

(CSS visual styling doesn't matter in terms of whether HTML markup is valid or not. :) )